devel/hyprlang/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        hyprwm hyprlang 0.5.0 v
+revision            0
+categories          devel
+license             LGPL-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         The hypr configuration language is an extremely efficient, \
+                    yet easy to work with, configuration language for applications.
+long_description    {*}${description}
+homepage            https://hyprland.org/hyprlang
+checksums           rmd160  5d309779fdf2e9d4037daf69c2569e714e9558e4 \
+                    sha256  c59e705f2c2ff9ea4e2b183fdf0bc20a62b7162604a657d6352716a1fd5061b2 \
+                    size    54809
+github.tarball_from archive
+
+patchfiles-append   0001-config.cpp-fix-environ-for-Apple.patch
+
+compiler.cxx_standard   2023
```

Comment:
are you sure this will do anything useful (i.e., is it actually defined in macports-base)?

Reply:
Well, it seems to pick the right compiler, but I can change it to 2020, sure.

Reply:
if it needs 2023 by all means leave it as-is. I was just expecting more manual blacklisting would be required as I doubt this standard is defined in macports-base yes. But admittedly, I didn't check that from my phone...

Reply:
How should we check? I do not know the threshold for Apple clangs for C++23 and probably have no OS matching it.

Perhaps let me copy from some portfile a needed blacklisting, it won’t hurt and we can be sure then.